plugins/out_es/es.c

```diff
@@ -62,8 +62,8 @@ static flb_sds_t add_aws_auth(struct flb_http_client *c,
     flb_http_add_header(c, "User-Agent", 10, "aws-fluent-bit-plugin", 21);
 
     signature = flb_signv4_do(c, FLB_TRUE, FLB_TRUE, time(NULL),
-                              ctx->aws_region, "es",
-                              0,
+                              ctx->aws_region, ctx->aws_service_name,
+                              S3_MODE_SIGNED_PAYLOAD,
```

Comment:
@matthewfala We need to test this with existing opensearch to ensure there is no regression.

Reply:
Actually I'm pretty sure you're always allowed to send this header, since it increases the security of the request

Reply:
https://docs.aws.amazon.com/opensearch-service/latest/developerguide/request-signing.html

This suggests all of opensearch supports it.